tracer/build/_build/MSBuildLogHelper.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Nuke.Common;
+using Nuke.Common.IO;
+
+using Logger = Serilog.Log;
+
+internal static class MSBuildLogHelper
+{
+    static internal AbsolutePath MsbuildDebugPath => NukeBuild.RootDirectory / "logs" / "msbuild";
```

Comment:
nit, I think we should put this in the `build_data` directory
```suggestion
    static internal AbsolutePath MsbuildDebugPath => TestLogsDirectory / "msbuild";
```